src/main/java/io/leonis/subra/game/data/TeamColor.java

```diff
@@ -0,0 +1,50 @@
+package io.leonis.subra.game.data;
+
+import java.awt.Color;
+
+/**
+ * The Enum TeamColor.
+ *
+ * This enumeration represents the possible colors a team can sport in a Small Size League
+ * sequence.
+ *
+ * @author Jeroen de Jong
+ * @author Rimon Oz
+ */
+public enum TeamColor {
+  /**
+   * Blue team color.
+   */
+  BLUE(Color.BLUE),
+  /**
+   * Yellow team color.
+   */
+  YELLOW(Color.YELLOW),
+  /**
+   * None team color.
+   */
+  NONE(Color.BLACK);
+
+  /**
+   * The color as a {@link Color}.
+   */
+  private Color color;
```

Comment:
Why do we need the Color as awt color? Seems rather specific.

Reply:
we don't. it used to be an awt.Color because of interop with ui that accepts awt.Color; it's not necessary now